app/license.go

```diff
@@ -5,21 +5,33 @@ package app
 
 import (
 	"bytes"
-	"errors"
 	"net/http"
 	"os"
 	"strings"
+	"time"
 
+	"github.com/dgrijalva/jwt-go"
 	"github.com/mattermost/mattermost-server/v5/mlog"
 	"github.com/mattermost/mattermost-server/v5/model"
 	"github.com/mattermost/mattermost-server/v5/utils"
+
+	"github.com/pkg/errors"
 )
 
 const (
-	requestTrialURL = "https://customers.mattermost.com/api/v1/trials"
-	LicenseEnv      = "MM_LICENSE"
+	requestTrialURL           = "https://customers.mattermost.com/api/v1/trials"
+	LicenseEnv                = "MM_LICENSE"
+	LicenseRenewalURL         = "https://customers.mattermost.com/subscribe/renew"
```

Comment:
Why remove the configuration for the URL? I believe it's a better approach than having this hardcoded here

Reply:
After talking with @srkgupta offline he suggested to better have it hardcoded to avoid the admins introduce a phishing URL or something similar.